roms_tools/setup/cdr_forcing.py

```diff
@@ -93,22 +131,27 @@ def check_unique_name(self) -> "ReleaseCollector":
 
         return self
 
-    def determine_release_type(self) -> "ReleaseCollector":
+    @model_validator(mode="after")
+    def check_all_releases_same_type(self):
         """Ensure all releases are of the same type, and set the release_type."""
-        release_types = set(map(type, self.releases))
+        release_types = set(map(type, self.root))
```

Comment:
```suggestion
        release_types = set(r.release_type for r in self.root)
```

Reply:
Tests were failing, but with this suggested change they are passing. (Annoying that the tests don't run on my fork.)